data_processors/pipeline/services/metadata_srv.py

```diff
@@ -195,3 +196,65 @@ def get_metadata_for_library_runs(library_runs: List[LibraryRun]) -> List[LabMet
     for lib_run in library_runs:
         library_id_list.append(lib_run.library_id)
     return get_metadata_by_keywords_in(libraries=library_id_list)
+
+
+def get_sorted_library_id_by_sequencing_time(library_ids: List[str]) -> str:
```

Comment:
If I understand the code correctly then this will/should return the most recent library ID of all the ones passed in to it? 
If so, I'd change the method name, `get_sorted_library_id_by_sequencing_time` sounds like you'll get a list of ids sorted by sequencing time. Perhaps something like `get_most_recent_library_id_by_sequencing_time`?

Reply:
Yes, it return most recent one. 
I will rename the function name as advised.